src/main/java/pl/shopgen/controllers/ProductPagination.java

```diff
@@ -0,0 +1,28 @@
+package pl.shopgen.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import pl.shopgen.models.Product;
+import pl.shopgen.services.ProductService;
+
+@RestController
+public class ProductPagination {
+
+    @Autowired
+    ProductService productService;
+
+    @RequestMapping(value = "/products",params = {"page","size"},method = RequestMethod.GET)
```

Comment:
Rozumiem, że po tym sprincie produkty będą się paginowały swoją drogą, a wyszukiwarka idzie swoją? Może łączymy to jezcze jakoś?
Czy można zrobić generyczną paginację?

Reply:
Wstępnie wstrzymam się z tym do czasu aż powstanie wyszukiwarka, wtedy zobaczę jak można to połączyć.